### DISCUSSION TITLE
BattleSnake Java starter code not working

### DISCUSSION THREAD

**Discussion by TomMitchell123** [BattlesnakeOfficial/feedback#234] [Bug Reports]
When you import the java starter code into replit and click run, it prints out many errors and does not run properly. Overall, it says there are 82 errors. Could someone help out with this issue?

**Comment by Nettogrof** [comment]
I just checked,  and it works correctly now.  I don't know what happen in March 2023

**Comment by xtagon** [comment]
Closing as resolved per @Nettogrof's comment, but feel free to re-open if that didn't fix it for ya!
